README.md

```diff
@@ -142,6 +142,33 @@ print(blaser_qe(src=src_embs, mt=mt_embs).item())  # 4.708
 Detailed model cards with more examples: [facebook/blaser-2.0-ref](https://huggingface.co/facebook/blaser-2.0-ref), 
 [facebook/blaser-2.0-qe](https://huggingface.co/facebook/blaser-2.0-qe). 
 
+### Classifying the toxicity of sentences with MuTox
+
+[MuTox](https://github.com/facebookresearch/seamless_communication/tree/main/src/seamless_communication/cli/toxicity/mutox), the first highly multilingual audio-based dataset with toxicity labels. The dataset consists of 20k audio utterances for English and Spanish, and 4k for the other 19 languages, and uses the multi-model and multilingual encoders from SONAR.
+
+```Python
+from sonar.models.mutox.loader import load_mutox_model
+from sonar.inference_pipelines.text import TextToEmbeddingModelPipeline
+
+t2vec_model = TextToEmbeddingModelPipeline(
+    encoder="text_sonar_basic_encoder",
+    tokenizer="text_sonar_basic_encoder",
+    device=device,
+)
+text_column='lang_txt'
+classifier = load_mutox_model(
+    "mutox",
```

Comment:
This code won't work unless we have a model card "mutox" in some place where fairseq2 model loader can access it. 
We have one such card [in the SC repo](https://github.com/facebookresearch/seamless_communication/blob/main/src/seamless_communication/cards/mutox.yaml), but we don't want the SONAR code to be dependent on it. 
So maybe we should copy the card to `sonar/cards`. And we might need to change the name of the new card; otherwise, there will be a model name onflict for those who have both SONAR and seamless_communication packages installed. 
@artemru how would you suggest to handle this gracefully?

Reply:
For now, I just added the card to `sonar/cards` but it might be best if it could be accessed through `https://dl.fbaipublicfiles.com/SONAR/`, but I do not have access to add it there. I guess another solution not to pull it from  `https://dl.fbaipublicfiles.com/seamless/models/mutox.pt` would be to add a release to this GitHub repo and add a section to download the model from there.

You tell me what's best.